query-node/schemas/membership.graphql

```diff
@@ -19,6 +39,9 @@ type MemberMetadata @entity {
 
   "Short text chosen by member to share information about themselves"
   about: String
+
+  "Social media handles, email address..."
+  externalResources: [MembershipExternalResource] @derivedFrom(field: "memberMetadata")
```

Comment:
I would suggest to use `[MembershipExternalResource!]!` or at least `[MembershipExternalResource!]` so we don't need with possible `null` values in the array

Reply:
Somehow: `[MembershipExternalResource] @derivedFrom(field: "memberMetadata")`
generates: `[MembershipExternalResource!]` in the `generated/graphql-server/generated/schema.graphql`
While `[MembershipExternalResource!]` -> `[MembershipExternalResource!]!` :confused:.

Most of the schema returns `null` instead of empty arrays so I just picked `[MembershipExternalResource!]` over `[MembershipExternalResource!]!` to stay consistent